### DISCUSSION TITLE
React 18 产生的 ReactDom.render() warning

### DISCUSSION THREAD

**Discussion by csc-bo** [alibaba/formily#3223] [Q&A 2.x] [answered]
FormDialog 和 FormDrawer 在react 18 都会有 [ Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. ]。 可以兼容下React 18吗？ 谢谢

**Comment by janryWang** [comment, marked as answer]
来个PR吧，就是判断下createRoot是否存在，如果存在优先使用createRoot
